3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,28 @@
+// Age-ify (A future age calculator)
+
+const yearOfBirth = 1987;
+const yearFuture = 2027;
+
+let age = yearFuture - yearOfBirth;
```

Comment:
using `const` instead of `let` would be better to go for, due to the fact that the value of `age` does not change.

Reply:
> using `const` instead of `let` would be better to go for, due to the fact that the value of `age` does not change.

Should I make changes or is this advice for the future?

Reply:
Well, that’s up to you!
Think about it, if it makes sense make the changes.
But I would recommend making changes 😊